### ISSUE TITLE
Module not found: Can't resolve 'recombee-js-api-client'

### ISSUE THREAD

**Issue by sharkfabri** [recombee/js-api-client#17]
I installed the module version 4.1.4 in my NextJs 14.0.3 project and I have this error: Module not found: Can't resolve 'recombee-js-api-client'

I tried to import in different ways than `import recombee from 'recombee-js-api-client'` and tried in a javascript project over my typescript one, but nothing seems to work.

**Comment by DanielBreiner** [comment]
Hi @sharkfabri, I wasn't able to reproduce the issue - the import works fine for me, both in JS and TS.

```js
import recombee from 'recombee-js-api-client'
const recombee = new recombee.ApiClient(/* ... */);
// or using named imports
import { ApiClient } from 'recombee-js-api-client'
const recombee = new ApiClient(/* ... */);
```

Could you please create and link a minimal reproduction repo?

**Comment by DanielBreiner** [comment]
Hi, I've managed to reproduce this - the issue was in the missing `main` field in package.json (or so I hope).
A fix landed in [v4.1.5](https://github.com/recombee/js-api-client/releases/tag/v4.1.5).
I will close this issue for now - please update the library and let me know if the issue persists.
